src/pages/CertificatesViewAll.js

```diff
@@ -20,8 +28,56 @@ const headersMap = {
     'Carbon Embodiment',
     'Status',
   ],
-  emma: ['Date', 'Commitment'],
-  reginald: [],
+  emma: [
+    'Date',
+    'Electric energy use',
+    'Production Start',
+    'Production End',
+    'Status',
+  ],
+  reginald: [
+    'Date',
+    'H2 Batch size',
+    'H2 Certificate holder',
+    'Carbon Embodiment',
+    'Status',
+  ],
+}
+
+const aggregateData = (data, id) => {
+  const rowsMap = {
+    heidi: (certs) =>
+      certs.map((cert) => [
+        <NameCell
+          key={cert.id}
+          date={formatDate(cert.created_at)}
+          name={formatCertName(cert)}
+        />,
+        `${cert.hydrogen_quantity_wh / 1000000} MWh`,
```

Comment:
Would it be better to add `.toFixed(n)` ?

Reply:
No

Reply:
I am sorry but for my understanding the idea of the ticket is to make it as close as possible to the design . 

if you add `.toFixed(1)` you will get something like `4.1 MWh` , if not you will get something like `4.12345678 MWh` which is not according to the design . 

see the design from Figma below:

<img width="676" alt="image" src="https://github.com/digicatapult/sqnc-hyproof-client/assets/58742260/77ed3186-475a-4f59-ae99-106ae02651f4">

Reply:
watt is an integer, it should always devide from 1 no?